src/test/scala/no/ndla/articleapi/UnitSuite.scala

```diff
@@ -28,7 +28,7 @@ abstract class UnitSuite extends FunSuite with Matchers with OptionValues with I
 
   setEnv("NDLA_BRIGHTCOVE_ACCOUNT_ID", "some-account-id")
   setEnv("NDLA_BRIGHTCOVE_PLAYER_ID", "some-player-id")
-  setEnv("SEARCH_ALIAS", "integration-test-index")
```

Comment:
De test indexene opprettes i beforeAll og slettes i afterAll() nettopp for å ungå det der.